src/plugins/ui_core_views/filter_evaluation.ts

```diff
@@ -74,6 +74,8 @@ export class FilterEvaluationPlugin extends UIPlugin {
         break;
       case "HIDE_COLUMNS_ROWS":
       case "UNHIDE_COLUMNS_ROWS":
+      case "ADD_COLUMNS_ROWS":
+      case "REMOVE_COLUMNS_ROWS":
```

Comment:
starting saas-16.1, the evauation of the cells is done in the 'finalize' step. If you want to rely on the latest evaluated values, you should recompute the hiddenrows in the finalize as well. you can look at how it's done for the UNDO/REDO commadns for instance :)
```suggestion
      case "REMOVE_COLUMNS_ROWS":
         this.isEvaluationDirty = true;
```

Reply:
Thank you for your guidance. I've implemented the changes :)

Reply:
hum, are you sure that this makes sense? you are modifying the behavior for HIDE/UNHIDE columns as well,  I don't think its necessary (or it should then be tested) why not moving the case for add/remove  above, where we already  set `isEvaluationDirty` to true?

Reply:
Yes, that makes sense, it should be good now